tests/lib/prepare-restore.sh

```diff
@@ -634,6 +634,11 @@ prepare_suite_each() {
         "$TESTSTOOLS"/cleanup-state pre-invariant
     fi
     tests.invariant check
+
+    SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
+    if os.query is-focal || os.query is.fedora; then
```

Comment:
is-fedora?

Reply:
Nice catch, I added all the classic systems now.